packages/camera/camera_android_camerax/test/resolution_selector_test.dart

```diff
@@ -0,0 +1,87 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:camera_android_camerax/src/aspect_ratio_strategy.dart';
+import 'package:camera_android_camerax/src/instance_manager.dart';
+import 'package:camera_android_camerax/src/resolution_selector.dart';
+import 'package:camera_android_camerax/src/resolution_strategy.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+
+import 'resolution_selector_test.mocks.dart';
+import 'test_camerax_library.g.dart';
+
+@GenerateMocks(<Type>[
+  TestResolutionSelectorHostApi,
+  TestInstanceManagerHostApi,
+])
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  group('ResolutionSelector', () {
+    tearDown(() {
+      TestResolutionSelectorHostApi.setup(null);
+      TestInstanceManagerHostApi.setup(null);
+    });
+
+    test('HostApi create creates expected ResolutionSelector instance', () {
```

Comment:
Same question, should this also have a corresponding `detached` create test with `verifyNever`